eth/beacon/helpers.py

```diff
@@ -208,63 +215,30 @@ def get_attestation_indices(crystallized_state: 'CrystallizedState',
 
 
 @to_tuple
-def get_active_validator_indices(dynasty: int,
-                                 validators: Iterable['ValidatorRecord']) -> Iterable[int]:
+def get_active_validator_indices(validators: Sequence['ValidatorRecord']) -> Iterable[int]:
     """
-    TODO: Logic changed in the latest spec, will have to update when we add validator
-    rotation logic.
-    https://github.com/ethereum/eth2.0-specs/commit/52cf7f943dc99cfd27db9fb2c03c692858e2a789#diff-a08ecec277db4a6ed0b3635cfadc9af1  # noqa: E501
+    Gets indices of active validators from ``validators``.
     """
-    o = []
-    for index, validator in enumerate(validators):
-        if (validator.start_dynasty <= dynasty and dynasty < validator.end_dynasty):
-            o.append(index)
-    return o
+    return [
```

Comment:
You should unroll this comprehension to use a `yield` pattern since you're using the `to_tuple` decorator.  Otherwise, you should convert this inner comprehension to be `tuple(i for thing in things)` so that we don't waste the overhead of constructiing the list and then converting to a tuple.

Reply:
So sorry for that I totally forgot to Py-EVM-ize this time(!!!). Inclined to use the second pattern (`tuple(i for thing in things)`) for syncing with the spec.